### ISSUE TITLE
Add an Accessibility option to allow the SearchBar (Omnibar) to stick at the bottom

### ISSUE THREAD

**Issue by sanogueralorenzo** [duckduckgo/Android#850]
Suggestion / enhancement:

As phones are getting bigger having interaction at the top of the screens can become uncomfortable (especially if it is a widget that has a lot of interaction).

Looking at how Kiwi has it implemented (it is a Chromium development option):

Under Settings -> Accessibility they have the following
![image](https://user-images.githubusercontent.com/5630683/83954505-a2052780-a7fe-11ea-96bd-2f7ce62b7121.png)

After enabling and upon user restart the bottom bar will stick at the bottom
![image](https://user-images.githubusercontent.com/5630683/83954515-c5c86d80-a7fe-11ea-8c65-186134a26057.png)

What do you think? Is it an enhancement that DDG could consider now that the bottom bar has been removed?

Thank you! 🙏

**Comment by seekr** [comment]
My I also emphasize the importance of the option of making the omnibar sticky? It can be distracting and frustrating to have UI appearing and disappearing as I scroll.

In long aspect ratio phones such as S8 I'm not gaining much by having a disappearing strip of UI.

Great work ddg team, as usual!

**Comment by blackICE19240** [comment]
I'd like to add that along with this it would be ideal to have the tab switcher stack tabs from the bottom too so you can reach them all with one hand.

**Comment by cmonfortep** [comment]
Update here: we recently had some discussion around this topic and we don't have plans to introduce a change on the omnibar position.

We know some folks like the toolbar at the bottom, but when we experimented with that, we didn't see a positive effect. That combined with a higher maintenance cost (UI widgets, dialogs, animations,... will need to support both variants), we discarded re-visiting this again (at least in the short/mid-term). 

However, that could change at some point if the team grows (🤞).
